packages/json-serializer/index.js

```diff
@@ -1,4 +1,23 @@
+/*
```

Comment:
wow , you actually worked on json stuff :) I thought we put it aside for now and I did not dare tackle it haha ;) well done

Reply:
Not really. I just changed the function signiture.